docs/DeveloperGuide.md

```diff
@@ -149,103 +149,104 @@ Classes used by multiple components are in the `seedu.storemando.commons` packag
-command. The `undo` command will call `Model#undoStoreMando()`, which will shift the `currentStatePointer` once to the
-left, pointing it to the previous inventory state, and restores the inventory to that state.
+![ReminderActivityDiagram](images/ReminderActivityDiagram.png)
 
-![UndoRedoState3](images/UndoRedoState3.png)
+#### Design consideration:
 
-<div markdown="span" class="alert alert-info">:information_source: **Note:** If the `currentStatePointer` is at index 0, pointing to the initial StoreMando state, then there are no previous StoreMando states to restore. The `undo` command uses `Model#canUndoStoreMando()` to check if this is the case. If so, it will return an error to the user rather
-than attempting to perform the undo.
+##### Aspect: How `reminder` executes
 
-</div>
+**Alternative 1 (current choice)** : provide integer as an input argument
+    * Pros: Faster to type as compared to date in a particular format.
+    * Cons: More cases to consider when parsing the commmand.
 
-The following sequence diagram shows how the undo operation works:
+**Alternative 2** : provide a date in the format of YYYY-MM-DD as input
+    * Pros: Easier to compare between items as the input date can be used to create an `expiryDate` object 
+            which can be used to compare with all the items' expiry dates.
+    * Cons: When the user wants to find items that are already expired, it is easier to key in a number then to 
+            find a particular date and key it in. This is more taxing on the user.
 
-![UndoSequenceDiagram](images/UndoSequenceDiagram.png)
 
-<div markdown="span" class="alert alert-info">:information_source: **Note:** The lifeline for `UndoCommand` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
+#### Clear Feature
 
-</div>
+##### Implementation
 
-The `redo` command does the opposite — it calls `Model#redoStoreMando()`, which shifts the `currentStatePointer` once to
-the right, pointing to the previously undone state, and restores the inventory to that state.
+This portion describes the implementation of the clear feature which allows users to clear items that are either in
+a particular location or clear all items in the list.
 
-<div markdown="span" class="alert alert-info">:information_source: **Note:** If the `currentStatePointer` is at index `storeMandoStateList.size() - 1`, pointing to the latest inventory state, then there are no undone StoreMando states to restore. The `redo` command uses `Model#canRedoStoreMando()` to check if this is the case. If so, it will return an error to the user rather than attempting to perform the redo.
+1. When the user keys in a command string, `execute` command of the `LogicManager` is called with the given string as input.
+2. In the method, `LogicManager` calls on the `parseCommand` method of StoreMandoParser to parse the given command.
```

Comment:
I think `StoreMandoParser` should have a grey highlight as mentioned above.

Reply:
It seems as though lines 3 to 7 have an additional space in front which you may have added due to the grey highlight. I think we might need to check if the additional space is actually needed.